setup.py

```diff
@@ -0,0 +1,45 @@
+import os.path
+import setuptools
+
+_here = os.path.dirname(__file__)
+
+with open(os.path.join(_here, 'README.md'), 'r', encoding='utf-8') as handle:
+    _long_description = handle.read()
+
+with open(os.path.join(_here, 'source', 'fab', '__init__.py'),
+          encoding='utf-8') as handle:
+    for line in handle:
+        bits = line.split('=', 1)
+        if bits[0].strip().lower() == '__version__':
+            _version = bits[1].strip()
+            break
+    else:
+        raise RuntimeError('Cannot determine package version.')
+
+setuptools.setup(
+    name='model-fab',
```

Comment:
Now that I see it written below I think I actually prefer `sci-fab`

Reply:
Happy to oblige.